### PR TITLE
/unstable

### PR DESCRIPTION
**Pull request by gurgeous**
How about this? Supports both `failureRate` and `seed`. See #53

### CODE REVIEW

File: httpbin/handlers.go
Comment (mccutchen) [inline httpbin/handlers.go:271]: For consistency with other optional parameters (e.g. `chunk_size`), let's call this `failure_rate`.

File: httpbin/handlers.go
Comment (mccutchen) [inline httpbin/handlers.go:1011]: Thanks for factoring out this functionality!  Would you mind moving this into `helpers.go` alongside the other small utility functions?

**Review by mccutchen — COMMENTED** [review]
This is awesome, thanks @gurgeous!  I have a couple of very minor requests below, but LGTM overall.

File: httpbin/handlers.go
Comment (gurgeous) [reply httpbin/handlers.go]: Ah, oversight on my part. Better?

File: httpbin/helpers.go
Comment (mccutchen) [inline httpbin/helpers.go:179]: Heh, in testing this locally I realized there's a (preexisting) issue with using `Unix()` here:  It has one second granularity, which leads to unexpectedly periodic behavior in this endpoint by default:

```
$ while true; do curl -v http://localhost:8080/unstable 2>&1 | grep '< HTTP'; sleep 0.2; done
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
^C
```

Changing this to `UnixNano()` leads to a better distribution of results:

```
$ while true; do curl -v http://localhost:8080/unstable 2>&1 | grep '< HTTP'; sleep 0.2; done
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 200 OK
< HTTP/1.1 200 OK
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
< HTTP/1.1 500 Internal Server Error
```

**Review by mccutchen — COMMENTED** [review]
Noticed one last thing to fix, if you don't mind.  Please feel free to squash all this down to one commit, and we can merge!

**Comment by gurgeous** [comment]
Ha! I will fix that, hang on

**Comment by gurgeous** [comment]
Oh - usually I hit Squash & Merge in lieu of manual squashing. Let me know if you prefer the manual squash. I don't do that often but I'm sure I can figure it out :)

**Review by mccutchen — APPROVED** [review]
👍 LGTM, thanks!  I'm actually unsure whether you'll have permission to do the squash & merge yourself once this is approved, but if so please do the honors.  If not, I'll do it for you.  (I figure if you are allowed to do it yourself, it'll more accurately attribute the commit to you.)

**Comment by gurgeous** [comment]
I think you have to merge the PR. No worries :) FYI, I believe you can configure the repo to force Squash & Merge for all PRs if that's what you prefer. We have our repos setup that way. Somewhere in settings. 

Thanks again for go-httpbin, we love it! Hopefully I'll have more opportunities to contribute.

**Comment by mccutchen** [comment]
> it'll more accurately attribute the commit to you

Thankfully, GitHub does the right thing here https://github.com/mccutchen/go-httpbin/commit/7ca56e77a65ba254196a8bf009139f258b4f62e5:

Screen Shot 2021-04-21 at 12 05 10 PM

**Pull request merged**
